lab_1/main.py

```diff
@@ -12,7 +11,18 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+
```

Comment:
You need checks for input arguments, smth like:
`if not isinstance(input_arg_name, expected_arg_type):`
`do smth here`.